test/supp/helgrind-test_ipc.supp

```diff
@@ -19,6 +19,7 @@
    [false-positive] Double check locking pattern in trackingOpenIpcHandle
    Helgrind:Race
    fun:utils_atomic_store_release_ptr
+   fun:upstreamOpenIPCHandle
```

Comment:
Wouldn't this disable suppression for false positive issues with a backtrace that lacks upstreamOpenIPCHandle? Shouldn't we use "..." here?

Reply:
But the `utils_atomic_store_release_ptr` is called only inside the `upstreamOpenIPCHandle` function.
Probably, I did not get your point.